numba_dppy/utils/type_conversion_fns.py

```diff
@@ -51,7 +51,11 @@ def npytypes_array_to_dppy_array(arrtype, addrspace=address_space.GLOBAL):
                              ``numba.types.Array``
 
     """
-    if isinstance(arrtype, types.npytypes.Array):
+    # We are not using isinstance() here as we want to
+    # strictly match with types.Array. There are Numba_dppy types
```

Comment:
Either use `(N)umba-dppy` capitalizing the `N` depending on where the word is used in the sentence, or if you want to refer to the actual Python module use `numba_dppy`. Under no circumstance should we use `Numba_dppy`.